internal/auth/backends/email_password/backend.go

```diff
@@ -27,12 +28,20 @@ func New(storeProvider auth.AccountStoreProvider, verifProvider verification.Pro
 	}
 }
 
+func isPasswordStrong(password string) bool {
+	return len(password) >= 10
+}
+
 func (e *EmailPasswordBackend) Register(ctx context.Context, namespace auth.AccountNamespace, credentials map[string]string) (*auth.Account, error) {
 	d, dErr := decode(credentials)
 	if dErr != nil {
 		return nil, dErr
 	}
 
+	if !isPasswordStrong(d.Password) {
+		return nil, fmt.Errorf("your password is weak. password should have at least 10 characters")
```

Comment:
Can this please wrap the `auth.ErrInvalidCredentials` error and update the message to just state the minimum password length.